### PR TITLE
Added slice plots for all 3 axes

### LINKED ISSUE #178: Graph all 3 slice axis as 2d plots
References task #176

Show 3 simultaneous plotly graphs, pulling from the 3d cube file data in different axes.

### PR DESCRIPTION
**Pull request by BraydenThompson**
Closes #178

## Discussion

Added all three ipyvolume slices within a single plot, each with an attached slider to control the selected slice. 

## Testing

- [ ] Run `build.sh` and make sure new dependencies install in .venv
- [ ] Open up the notebook (make sure using the .venv Python kernel)
- [ ] Execute the first cell, and ensure that the all three slices show up beneath the ipyvolume graph, and each of the sliders updates the corresponding graph. 
image

### CODE REVIEW

**Review by ryliewalsh — COMMENTED** [review]
Works well, however the display is always on and seems to skew the main menu. I can try and and mess with the visibility for clean up.

**Review by matthew-oberg — APPROVED** [review]
Functional, but agree that the UI isn't great. Approved on the condition that the PR which links the 3D and 2D sliders cleans it up

**Review by ryliewalsh — APPROVED** [review]

File: GoVizzy/GoVizzy.ipynb
Comment (ryliewalsh) [inline GoVizzy/GoVizzy.ipynb:1]: "%run Main.py\n"
 "%matplotlib widget\n",
   
swapping the order fixes the UI issue (:

**Review by dignojrteogalbo — APPROVED** [review]
I can see the 3 different plots and each of the sliders adjusts them accordingly.

**Pull request merged**